.circleci/config.yml

```diff
@@ -98,7 +98,7 @@ jobs:
         type: string
       xcode:
         type: string
-        default: "14.2.0"
+        default: "16.0.0"
```

Comment:
I really wish we could just do `xcode: "latest"` for these pure Python packages.. but I don't see anything like that in the CircleCI configuration reference. I wonder if it would support `xcode: "16"` or something.

Reply:
The docs are pretty explicit about allowed version numbers, but I'll test it.

Reply:
poking around their reference a bit more, you might be able to do
```
machine: xcode
```
see https://circleci.com/developer/images?imageType=machine

Reply:
So, `16`, is not supported (image `xcode:16` not found), `"16"` fails because CircleCI's string-to-number parser requires at least a minor version (regex: `^\d+(\.\d+){1,2}(-[a-zA-Z]+)?$`), and `"16.0"` works, but it resolves to `"16.0.0"`, which is not helpful at all.

One workaround that comes to mind is by introducing another layer of indirection (a solution to any problem in CS) -- in this case `orb-ocean`. In `orb-ocean` we have a `macos` executor that has a default `xcode`, so this would work:
```
orbs:
  ocean: dwave/ocean@1

jobs:
  test-macos:
    executor:
      name: ocean/macos
    steps:
      ...
```

Reply:
[`machine` executor](https://circleci.com/docs/configuration-reference/#machine) seems to only support Linux, Android and Windows images.